test/week_test.js

```diff
@@ -57,6 +57,44 @@ describe('Week', () => {
     assert(day.prop('day').isSame(dayClicked, 'day'))
   })
 
+  it('should call the provided onWeekSelect function', () => {
+    let firstDayReceived = null
+    let weekNumberReceived = null
+
+    function onWeekClick (day, newWeekNumber) {
+      firstDayReceived = day.format()
+      weekNumberReceived = newWeekNumber
+    }
+
+    const weekStart = moment('2015-12-20')
+    const realWeekNumber = parseInt(weekStart.format('w'), 10)
+    const week = mount(
+      <Week day={weekStart} showWeekNumber onWeekSelect={onWeekClick} />
+    )
+    const weekNumberElement = week.find(WeekNumber)
+    weekNumberElement.simulate('click')
+    expect(firstDayReceived).to.equal(weekStart.format())
+    expect(weekNumberReceived).to.equal(realWeekNumber)
```

Comment:
this two expectations are orthogonal so should be in separate tests

(this applies to the 'should set the week number with the provided formatWeekNumber function' test too)